.travis.yml

```diff
@@ -21,28 +20,28 @@ matrix:
   include:
   - compiler: gcc
     os: linux
-    env: TASK="build" DISTRO=xenial
+    env: TASK="build" DOCKER_IMAGE=mrpt/build-env-full:latest
   - compiler: gcc
     os: linux
-    env: TASK="test" DISTRO=xenial
+    env: TASK="test" DOCKER_IMAGE=mrpt/build-env-full:latest
   - compiler: gcc
     os: linux
-    env: TASK="build" DEPS="headless" DISTRO=xenial
+    env: TASK="build" DEPS="headless" DOCKER_IMAGE=mrpt/build-env-full:latest
```

Comment:
The headless and minimal probably won't do what you expect with the new image because it already has everything installed.